en/user-guide/layerexplanations/layerexplanation.rst

```diff
@@ -27,33 +32,33 @@ Main layer for line-wastewater structures (channels). Creating a new record in t
 Wastewater Structures
 ---------------------
 
-od_wastewater_structure
-^^^^^^^^^^^^^^^^^^^^^^^
+Wastewater Structures (od_wastewater_structure)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This layer shows the detailed geometries of wastewater structures. You digitize a new detailed geometry by the records of the layer vw_qgep_wastewater_structure. If you want to change an existing geometry, then you have to edit in the layer od_wastewater_structure. In database-schema qgep_od.
 
-Structure Parts
-^^^^^^^^^^^^^^^
+Structure Parts (od_structure_part)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Cover is the only structure part with a point-geometry himself. All other structure parts are just linked to there wastewater structures and should by only edited by the main layers (vw_qgep_wastewater_structure and vw_qgep_reach).
```

Comment:
Cover is the only structure part with a point-geometry itself.

Reply:
Fixed